### DISCUSSION TITLE
Is this project active?

### DISCUSSION THREAD

**Discussion by ghost** [arturo-lang/arturo#938] [Q&A] [answered]
Hi.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by drkameleon** [comment, marked as answer]
First of all, hi and welcome! 😄 

Regarding your question, I think all you have to do is have a look at the issues and PRs being processed on a daily basis; I think we're going pretty well 😉 

Feel free to let me know in case you need any guidelines, but - in any case - you are obviously more than welcome to contribute, in any way!

**Reply by ghost** [reply]
Hi drkameleon.

> First of all, hi and welcome! 😄
> Regarding your question, I think all you have to do is have a look at the issues and PRs being processed on a daily basis; I think we're going pretty well 😉
> Feel free to let me know in case you need any guidelines, but - in any case - you are obviously more than welcome to contribute, in any way!

thank you for feedback.  *Have a great day, and thank you for your patience in answering my question.*

**Reply by drkameleon** [reply]
Whenever you need anything, we're here.

For general questions, our [channel on Discord](https://discord.gg/YdVK2CB) is quite appropriate for this type of thing.

For development instructions, I've done my best to make it as easy to contribute as possible, but still... I'm open for questions. @RickBarretto, who - despite his well-deserved holiday break these days - has been very active in Arturo's development lately, could also give a helping hand I guess 😉 

Have a great day as well! 😄

**Reply by ghost** [reply]
Hi drkameleon.

> Whenever you need anything, we're here.
> For general questions, our [channel on Discord](https://discord.gg/YdVK2CB) is quite appropriate for this type of thing.
> For development instructions, I've done my best to make it as easy to contribute as possible, but still... I'm open for questions. @RickBarretto, who - despite his well-deserved holiday break these days - has been very active in Arturo's development lately, could also give a helping hand I guess 😉
> Have a great day as well! 😄

thank you for feedback again. So... you are incredible.

**Reply by RickBarretto** [reply]
Hi, and welcome! 
I'm happy to see more people interested on Arturo's development. 🙂 

> Is this project active?

Yes, it is! Even these are the statistics of the project from this last month:
![image](https://user-images.githubusercontent.com/78623871/209202562-cfe76c56-a38d-4e46-a5a6-b50294997763.png)
[source](https://github.com/arturo-lang/arturo/pulse/monthly)
